src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx

```diff
@@ -76,9 +85,9 @@ function getLeakedPasswords({
     breachSite: securityQuestionBreachSite,
   } = getBreachInfo(unresolvedSecurityQuestionsBreach);
 
-  const leakedPasswordsData: LeakedPassword[] = [
+  const leakedPasswordsData: LeakedPasswordsLayout[] = [
     {
-      type: "password",
+      type: "passwords",
```

Comment:
We’ll need to change the navigation path in `.storybook/preview.tsx` as well here to not break it here: https://github.com/mozilla/blurts-server/blob/mntor-2259-steps/.storybook/preview.tsx#L158.